client/src/plus/deep-dives/byline/index.tsx

```diff
@@ -26,13 +27,13 @@ export function Byline({
       <div className="author-byline">
         <h3 className="author-name">{`${author}, ${authorDescription}`}</h3>
 
-        {authorBioURLS && (
+        {authorBioURLs && authorBioURLs.length > 0 && (
           <ul className="author-links">
-            {authorBioURLS.map((url, index) => {
+            {authorBioURLs.map(({ url, text }) => {
               return (
-                <li>
+                <li key={url}>
```

Comment:
Doh!

Reply:
I do wonder...
The reason that warnings isn't breaking our builds is because we don't test our React code in dev mode in CI. 
Well, we do but then you can't get React to "crash". But even if it did, how could you possibly know. We run a headless server tha clicks around on *some* URLs on `localhost:3000` but we don't do any inspection of the console.log output in Devtools.